components/form/form.css

```diff
@@ -0,0 +1,38 @@
+.content {
+  margin-top: 40%;
```

Comment:
I just fiddled with it abit on the browser element tools and this line seems to be causing the issue!
### Set it to `margin-top: 120px;` (or other desired stable unit size value) instead of `40%`
---
https://github.com/Katemuya/Recap-Project-2-Quiz-App/assets/31593501/efb1bc2d-56e9-4f68-806c-49b1cc423f9a

Reply:
I used em instead. That way the content will scale accordingly :)

Reply:
Thanks alot!